tools-v2/pkg/cli/command/curvebs/snapshot/copyset/copyset.go

```diff
@@ -56,97 +60,202 @@ func (sRpc *SnapshotRpc) Stub_Func(ctx context.Context) (interface{}, error) {
 	return sRpc.Client.Snapshot(ctx, sRpc.Request)
 }
 
-type SnapshotOneCommand struct {
+type SnapshotAllRpc struct {
+	Info    *basecmd.Rpc
+	Request *cli2.SnapshotAllRequest
+	Client  cli2.CliService2Client
+}
+
+func (sRpc *SnapshotAllRpc) NewRpcClient(cc grpc.ClientConnInterface) {
+	sRpc.Client = cli2.NewCliService2Client(cc)
+}
+
+func (sRpc *SnapshotAllRpc) Stub_Func(ctx context.Context) (interface{}, error) {
+	return sRpc.Client.SnapshotAll(ctx, sRpc.Request)
+}
+
+type SnapshotCopysetCommand struct {
 	basecmd.FinalCurveCmd
 
-	Rpc      *SnapshotRpc
-	Response *cli2.SnapshotResponse2
-	row      map[string]string
+	Rpc     *SnapshotRpc
+	AllRpcs []*SnapshotAllRpc
+	needAll bool
+	rows    []map[string]string
+
+	timeout    time.Duration
```

Comment:
this two args just use for rpc init, can delete it in class define, and add it to ppc init function (such as `doAllSnapshotsInit`)
```diff
- timeout    time.Duration
- retryTimes int32
```

Reply:
done